ipa-core/src/protocol/ipa_prf/boolean_ops/share_conversion_aby.rs

```diff
@@ -412,14 +391,9 @@ mod tests {
                             |idx, chunk| {
                                 let ctx = ctx.clone();
                                 async move {
-                                    let mut match_keys_builder = XS::builder();
-                                    for _ in 0..MK_BITS {
-                                        match_keys_builder
-                                            .push(AdditiveShare::<Boolean, CHUNK>::ZERO);
-                                    }
-                                    let mut match_keys = match_keys_builder.build();
+                                    let mut match_keys = BitDecomposed::new(iter::empty());
                                     match_keys.transpose_from(&chunk).unwrap_infallible();
```

Comment:
Unrelated to this PR... but just as an aside, I'm not sure why the `transpose_from` API is shaped like this. Why pass in a mutable `BitDecomposed`? In all of the instances I've seen recently, we just initialize an empty `BitDecomposed`.